packages/web-fragments/src/gateway/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
 	"extends": "../../tsconfig.base.json",
 	"compilerOptions": {
-		"lib": ["DOM", "ESNext"]
+		"lib": ["DOM", "ESNext"],
+		"allowImportingTsExtensions": true
```

Comment:
curious... why is this a good thing?

Reply:
The PR still needs some work to improve. I suspect this can go away but I was dealing with module import type ambiguity issues and allowing .ts imports allowed me past so I could focus on the rest